src/test/suite/fixtures/recorded/compoundTargets/clearJustThisPastEndOfToken.yml

```diff
@@ -0,0 +1,36 @@
+languageId: plaintext
+command:
+  spokenForm: clear just this past end of token
+  version: 3
+  targets:
+    - type: range
+      anchor:
+        type: primitive
+        modifiers:
+          - {type: toRawSelection}
+        mark: {type: cursor}
+      active:
+        type: primitive
+        modifiers:
+          - {type: position, position: end}
+          - type: containingScope
+            scopeType: {type: token}
+      excludeAnchor: false
+      excludeActive: false
+  usePrePhraseSnapshot: true
+  action: {name: clearAndSetSelection}
+initialState:
+  documentContents: hello there
+  selections:
+    - anchor: {line: 0, character: 8}
+      active: {line: 0, character: 8}
+  marks: {}
+finalState:
+  documentContents: hello th
+  selections:
+    - anchor: {line: 0, character: 8}
+      active: {line: 0, character: 8}
+fullTargets: [{type: range, excludeAnchor: false, excludeActive: false, rangeType: continuous, anchor: {type: primitive, mark: {type: cursor}, modifiers: [{type: toRawSelection}]}, active: {type: primitive, mark: {type: cursor}, modifiers: [{type: containingScope, scopeType: {type: token}}], positionModifier: {type: position, position: end}}}]
+ide:
+  messages:
+    - {type: warning, id: deprecatedPositionInference}
```

Comment:
how come we're using this deprecated command form?  presumably we should switch to "past end of its token"?

Reply:
Doesn't matter for this test.